src/js/actions/tools.js

```diff
@@ -791,6 +792,39 @@ define(function (require, exports) {
             fn: _vectorMaskHandler
         });
 
+        _vectorMaskPolicyHandler = function () {
```

Comment:
The function itself is what you might debounce, to get a new function which is then made the UIStore change event handler.